forge-gui/res/cardsfolder/upcoming/ascend_from_avernus.txt

```diff
@@ -0,0 +1,9 @@
+Name:Ascend from Avernus
+ManaCost:X W W W
+Types:Sorcery
+A:SP$ ChangeZoneAll | Origin$ Graveyard | Destination$ Battlefield | ChangeType$ Creature.YouOwn+cmcLEX,Planeswalker.YouOwn+cmcLEX | SubAbility$ DBChange | SpellDescription$ Return all creature and planeswalker cards with mana value X or less from your graveyard to the battlefield. Exile CARDNAME.
```

Comment:
I think you should trim `Exile CARDNAME.` from this SpellDescription$ and add it to a SpellDescription in line 5

Reply:
Moved `Exile CARDNAME` clause from line 4 to line 5